### PR TITLE
Feature/data table add form

### PR DESCRIPTION
**Pull request by Viserion77**
This pr just add PbFrom to PbDataTable

### CODE REVIEW

File: src/components/DataVisualization/DataTable/EntityController.vue [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        </h3>
+      </template>
+      <template #main>
+        <PbForm
+          v-model="dataTableController.entitySelected"
+          :entity-schema="entitySchema"
+        />
+      </template>
+      <template #footer>
+        <div style="display: flex; justify-content: right">
```
Comment (planivic1) [inline src/components/DataVisualization/DataTable/EntityController.vue:20]: class

**Review by planivic1 — CHANGES_REQUESTED** [review]

**Comment by mickaelnn** [comment]
Print?

**Comment by Viserion77** [comment]
image

**Review by ixpd — APPROVED** [review]
LGTM

**Review by marcosalves-bitg — APPROVED** [review]
LGTM

**Review by planivic1 — APPROVED** [review]
LGTM

**Pull request merged**